examples/lotr/src/main/scala/com/raphtory/examples/lotr/TutorialRunner.scala

```diff
@@ -24,7 +24,7 @@ object TutorialRunner extends App {
   val url  = "https://raw.githubusercontent.com/Raphtory/Data/main/lotr.csv"
   FileUtils.curlFile(path, url)
 
-  val graph   = Raphtory.newGraph()
+  val graph   = Raphtory.connect("raphtory").newGraph()
```

Comment:
can just set this back  (or leave) everyone messing with this file at the moment